### PR TITLE
[MU4] UI fractional scaling

### PR DESCRIPTION
**Pull request by Eism**
Resolves: #9929

### CODE REVIEW

**Comment by MarcSabatella** [comment]
I don't know what "fractional scaling" is, but does this by chance address the fact that the icons in MU4 and the score itself are way too small on high DPI displays, and that unlike previous releases, the -D command line option can't be used to override this?  This to me is a showstopper issue, as it makes MU4 all but unusable for me right now.  And I'd really like to be able to start moving forward soon on documentation.  See https://github.com/musescore/MuseScore/issues/10354

**Comment by Eism** [comment]
Hey @MarcSabatella!
No, this PR is not fixed #10354. 
I saw your issue and wanted to fix it, but I just forgot about it :( Sorry

So, I tried fixing the custom DPI setting, but that caused a few problems that take time to investigate. Therefore, I will create a separate PR with a fix.

**Review by RomanPudashkin — APPROVED** [review]

**Comment by DmitryArefiev** [comment]
Tested #9929 on Win10 - **Now incremental scaling in Windows is FIXED**  
But also a bug with score blurring appears now (I will log it a separate issue)

File: src/appshell/appshell.cpp [hunk 1]
```diff
@@ -83,6 +83,15 @@ int AppShell::run(int argc, char** argv)
 
     QCoreApplication::setAttribute(Qt::AA_EnableHighDpiScaling);
 
+    //! NOTE: For unknown reasons, Linux scaling for 1 is defined as 1.003 in fractional scaling.
+    //!       Because of this, some elements are drawn with a shift on the score.
+    //!       Let's make a Linux hack and round values above 0.75(see RoundPreferFloor)
+#ifdef Q_OS_LINUX
+    QGuiApplication::setHighDpiScaleFactorRoundingPolicy(Qt::HighDpiScaleFactorRoundingPolicy::RoundPreferFloor);
```
Comment (theofficialgman) [inline src/appshell/appshell.cpp:90]: @Eism please provide more information. this change breaks fractional scale values. if you have a particular issue with one DE and display server combination, it is best to have it corrected upstream and not break this for everyone else.

with the current code we get:
1X scaling for 0.75-1.749999
2X scaling for 1.75-2.749999
3X scaling for 2.75-3.749999
etc

this is not a good system. using passthrough would be much better.

**Pull request merged**